yang-packages/draft-ietf-netmod-yang-packages.xml

```diff
@@ -1137,23 +1124,6 @@ module ietf-yang-package-types {
          software packages, or hot fixes.";
     }
     
-    leaf previous-version {
-      type pkg-version;
-      description
-        "The previous package version that this version has been
-         derived from.  This leaf allows a full version history graph
-         to be constructed if required.";
-    }
-
-    leaf nbc-changes {
-      type boolean;
-      default false;
-      description
-        "Indicates whether the defined package version contains
-         non-backwards-compatible changes relative to the package
-         version defined in the 'previous-version' leaf.";
-    }
-
     leaf-list tag {
       type tags:tag;
       description
```

Comment:
In the "list included-package {" below, change this sentence:

         If included packages implement different revisions or
         versions of the same module, then an explicit entry in the

to this (drop the "or versions"):

         If included packages implement different revisions 
         of the same module, then an explicit entry in the